apps/vr-tests/src/stories/PivotNext.stories.tsx

```diff
@@ -176,7 +176,7 @@ storiesOf('Pivot Next Overflow', module)
     </Pivot>
   ))
   .addStory('Tabs - RTL', () => (
-    <Fabric dir="rtl">
+    <ThemeProvider dir="rtl" theme={{ rtl: true }}>
```

Comment:
can remove `dir="rtl"` once this is merged: https://github.com/microsoft/fluentui/pull/15347